Formula/ogre2.1.rb

```diff
@@ -4,6 +4,7 @@ class Ogre21 < Formula
   url "https://bitbucket.org/sinbad/ogre/get/06a386fa64e79a7204a90faf53da1735743f6c2e.tar.bz2"
   version "2.0.99999~pre0~0~20180616~06a386f"
   sha256 "d2e28bfcfbb1277355047c1d8bcd141b05b83af52d277725168e4281eac92a6d"
+  revision 1
```

Comment:
I think that this change is making the CI to fail

Reply:
that makes travis and azure fail because the bottle hasn't been built yet

once a bottle is built the CI should be happy

Reply:
I'm just having troubles getting a bottle to build successfully